services_app/src/main/AndroidManifest.xml

```diff
@@ -110,7 +110,7 @@
         <activity
             android:name=".MainActivity"
             android:label="@string/app_name"
-            android:theme="@style/ODKX">
+            android:theme="@style/SplashTheme">
```

Comment:
Why change the complete Services style?

Reply:
The [Android documentation](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#:~:text=In%20the%20manifest%2C%20replace%20the%20theme%20of%20the%20starting%20activity%20to%20the%20theme%20you%20create%20in%20the%20previous%20step.) recommends setting the Main Activity's theme to the one created specifically for the Splash Screen

This is required because the Splash Screen theme must be applied to the Main Activity to display the Splash Screen when the app launches. After the Splash Screen is dismissed, the Main Activity’s theme will automatically switch to `@style/ODKX`, as defined by the `postSplashScreenTheme` attribute within the `@style/SplashScreen` theme.